File: flutter_cab_pwa/lib/models/request_login.dart
// models/request_login.dart

class RequestLogin {
  final String? webLogin;
  final String? password;

  RequestLogin(this.webLogin, this.password);

  Map<String, dynamic> toJson() {
    return {
      "web_login": webLogin,
      "password": password
    };
  }
}

File: CabPwaApi/CabPwaApi/Requests/RequestLogin.cs
// Requests/RequestLogin.cs
using System.Text.Json.Serialization;

namespace CabPwaApi.Requests
{
    public class RequestLogin
    {
        [JsonPropertyName("web_login")]
        public string web_login { get; set; } = string.Empty;

        [JsonPropertyName("password")]
        public string password { get; set; } = string.Empty;
    }
}
